sklearn/feature_selection/rfe.py

```diff
@@ -146,6 +149,13 @@ def fit(self, X, y):
 
         support_ = np.ones(n_features, dtype=np.bool)
         ranking_ = np.ones(n_features, dtype=np.int)
+
+        if step_score:
+            # Determine the number of subsets of features
+            self.scores_ = np.zeros(
```

Comment:
Oh, I mean. First, ```scores``` in RFECV has to be an array to do ```scores += rfe.scores_``` in L435. It cannot a list in RFECV. If ```rfe.scores_``` is a list, I have to reverse the list and create a new array in L435 in RFECV. Second, ```scores``` RFECV is just the same as in RFE, it needs to compute the size beforehand anyway as well in L417. So I just do it in RFE as well, L155.  I am OK to remove the computation, actually it is not so complicated....